samples/graph-activity-feed-broadcast/nodejs/Manifest/manifest.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
  "manifestVersion": "1.12",
   "version": "1.0.0",
-  "id": "<<YOUR-MANIFEST-ID>>",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
```

Comment:
It's not compulsory to always use <<YOUR-MICROSOFT-APP-ID>>
It can be any Guid, so it's better to put like this <<YOUR-MANIFEST-ID>> and mention in readme that it can be any Guid or your app id.